src/styles/main.scss

```diff
@@ -1,3 +1,15 @@
-body {
-  margin: 0;
+@import "utils/variables.scss";
+
+@import "blocks/page.scss";
+@import "blocks/calendar.scss";
+
+@import "utils/reset.scss";
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
 }
```

Comment:
You are using this after you already used in the imported css. It works but maybe mentors can explain us why :) This is just a comment, you don't need to do anything. I think mentors can clarify this for us.